pkg/model/constants.go

```diff
@@ -2,26 +2,31 @@ package model
 
 // Constants for a community Keycloak installation
 const (
-	ApplicationName                    = "keycloak"
-	MonitoringKey                      = "middleware"
-	DatabaseSecretName                 = ApplicationName + "-db-secret"
-	PostgresqlPersistentVolumeName     = ApplicationName + "-postgresql-claim"
-	PostgresqlDeploymentName           = ApplicationName + "-postgresql"
-	PostgresqlDeploymentComponent      = "database"
-	PostgresqlServiceName              = ApplicationName + "-postgresql"
-	PostgresqlImage                    = "postgres:9.5"
-	KeycloakImage                      = "quay.io/keycloak/keycloak:7.0.1"
-	RHSSOImage                         = "registry.access.redhat.com/redhat-sso-7/sso73-openshift:1.0"
-	KeycloakDiscoveryServiceName       = ApplicationName + "-discovery"
-	KeycloakDeploymentName             = ApplicationName
-	KeycloakDeploymentComponent        = "keycloak"
-	PostgresqlDatabase                 = "root"
-	PostgresqlPersistentVolumeCapacity = "1Gi"
-	DatabaseSecretUsernameProperty     = "user"
-	DatabaseSecretPasswordProperty     = "password"
-	KeycloakServicePort                = 8443
-	AdminUsernameProperty              = "ADMIN_USERNAME"        // nolint
-	AdminPasswordProperty              = "ADMIN_PASSWORD"        // nolint
-	ServingCertSecretName              = "sso-x509-https-secret" // nolint
-	RouteLoadBalancingStrategy         = "source"
+	ApplicationName                     = "keycloak"
+	MonitoringKey                       = "middleware"
+	DatabaseSecretName                  = ApplicationName + "-db-secret"
+	PostgresqlPersistentVolumeName      = ApplicationName + "-postgresql-claim"
+	PostgresqlDeploymentName            = ApplicationName + "-postgresql"
+	PostgresqlDeploymentComponent       = "database"
+	PostgresqlServiceName               = ApplicationName + "-postgresql"
+	PostgresqlImage                     = "postgres:9.5"
+	KeycloakImage                       = "quay.io/keycloak/keycloak:7.0.1"
+	KeycloakInitContainerImage          = "quay.io/integreatly/sso_plugins_init:2.0.0"
```

Comment:
@pb82 How close are you to migrating the repo over? If it will be done tomorrow, would be good to switch this over.

Reply:
I'll update this on Monday

Reply:
Thanks @pb82